src/tests/execute/create_ask_tests.rs

```diff
@@ -63,13 +64,14 @@ mod create_ask_tests {
             },
         );
 
-        deps.querier.with_attributes(
-            "asker",
-            &[
-                ("ask_tag_1", "ask_tag_1_value", "String"),
-                ("ask_tag_2", "ask_tag_2_value", "String"),
-            ],
-        );
+        // TODO: find alternative function
+        // deps.querier.with_attributes(
```

Comment:
same, search "with_attributes" in https://github.com/FigureTechnologies/digital-currency-consortium/pull/159/files, there is example